milestones/migrations/0002_seed_relationship_types.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+from milestones.data import RELATIONSHIP_TYPES
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """
+        Adds database entries for milestone relationship types defined in data.py
+        """
+        for name in RELATIONSHIP_TYPES.values():
+            orm.MilestoneRelationshipType.objects.create(
+                name=name,
+                description='Autogenerated milestone relationship type "{}"'.format(name),
+            )
+
+    def backwards(self, orm):
+        """
+        A backwards migration of seeded MilestoneRelationshipType data is difficult due to the
+        likelihood of foreign key dependencies established over time.  So for now we'll raise an
+        error and request a manual clean-up of these relationships.
+        """
+        raise RuntimeError("Cannot reverse MilestoneRelationshipType seed migration.  Please clean up all dependencies manually.")
```

Comment:
I agree with @chrisndodge -- you can make `backwards` a noop and make forwards only create the entries if they don't already exist.